var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -3,8 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
 from spack import *
+import os
 
 
 class Legion(CMakePackage):
```

Comment:
Have you tried inheriting `CudaPackage` for the variants and conflicts it defines? You can inherit multiple packages like:

```
class Legion(CMakePackage, CudaPackage):
```

Reply:
Nope, I'm still learning!

I would certainly like to look more closely at this but would like to roll it into the next big update on tap to add configuration and build support for AMD/HIP (which landed in Legion just a few weeks ago).  I'd like to get this upstreamed for the flecsi folks before I dive into these changes. 

Would (more?) multiple inheritance via HIPPackage the path to take here too?

Reply:
Addressing that in a later PR sounds good to me!